packages/react-jss/tests/types/createUseStyles.ts

```diff
@@ -172,53 +171,14 @@ const noThemeArg4ClassesPass2 = noThemeArg4({...expectedCustomProps, theme: expe
 const noThemeArg5 = createUseStyles<string, MyProps, MyTheme>({
   singleNest: {
     fontWeight: 'bold',
-    singleValue: ({property, theme}) => '',
-    nestOne: ({property, theme}) => ({
+    // @ts-expect-error
+    nestOne: ({property, theme}: MyProps & {theme: MyTheme}) => ({
       color: 'red',
-      // @ts-expect-error
-      nothingAllowed: ({theme: innerTheme, ...innerProps}) => ''
+      display: () => 'block'
     })
   }
 })
 
-// Nested declarations are banned (double nest test)
-const noThemeArg6 = createUseStyles<string, MyProps, MyTheme>({
-  doubleNest: {
-    fontWeight: 'bold',
-    singleValue: ({property, theme}) => '',
-    firstNest: {
-      color: 'red',
-      innerSingleValue: ({property, theme}) => '',
-      secondNest: ({property, theme}) => ({
-        backgroundColor: 'blue',
-        // @ts-expect-error
-        nothingAllowed: ({theme: innerTheme, ...innerProps}) => ''
-      })
-    }
-  }
-})
-
-// Nested declarations are banned (triple nest test)
```

Comment:
why would one need double nest and triple nest test, is there code that is explicitly working in one of them and not in another?